hail/python/test/hail/helpers.py

```diff
@@ -130,7 +130,9 @@ def wrapper(func, *args, **kwargs):
 
 
 def assert_evals_to(e, v):
-    assert hl.eval(e) == v
+    res = hl.eval(e)
```

Comment:
does pytest not generate a nice error here?

Reply:
it didn't for me. Don't know if that was some sort of setting thing.